ts/editable/Mathjax.svelte

```diff
@@ -25,13 +25,17 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     import { pageTheme } from "$lib/sveltelib/theme";
 
     import { convertMathjax, unescapeSomeEntities } from "./mathjax";
+    import { ChangeTimer } from "../editor/change-timer";
```

Comment:
While not a problem at the moment as we have a single compilation unit, this is introducing a circular dependency. change-timer would be better moved to editable or something like sveltelib.